### ISSUE TITLE
check message signature not working

### ISSUE THREAD

**Issue by therealssj** [skycoin/hardware-wallet-go#77]
**Describe the bug**
`checkMessageSignature` does not work

**Environment information:**

- OS: any

**Steps to Reproduce**

```
skycoin-hw-cli checkMessageSignature --address=2EU3JbveHdkxW6z5tdhbbB2kRAWvXC2pLzw --message="Hello World" --signature=GvKS4S3CA2YTpEPFA47yFdC5CP3y3qB18jwiX1URXqWQTvMjokd3A4upPz4wyeAyKJEtRdRDGUvUgoGASpsTTUeMn
```

**Actual behavior**
`Action cancelled by user`

**Expected behavior**
```
Success 2! address that issued the signature is: 
#2EU3JbveHdkxW6z5tdhbbB2kRAWvXC2pLzw
```

**Comment by stdevAlDen** [comment]
@therealssj I'm going to finish #51 today, this is a possible duplicate of...

**Comment by therealssj** [comment]
@stdevAlDen okay but leave this issue open im not really sure if its a possible duplicate

**Comment by stdevAlDen** [comment]
@therealssj & @olemis 
I found the bug here: https://github.com/skycoin/hardware-wallet-go/blob/master/src/device-wallet/messages.go#L72

Currently solved here:
https://github.com/simelo/skycoin-hardware-wallet-go/blob/stdevAlDen_t51_check_message_signature_response/device-wallet/device-wallet.go#L151

I can finish it when scheduled

**Comment by therealssj** [comment]
lol shit XD

**Comment by olemis** [comment]
> @therealssj & @olemis
> I found the bug here: https://github.com/skycoin/hardware-wallet-go/blob/master/src/device-wallet/messages.go#L72
> 

Is this file generated from protobuffer specs ?

**Comment by olemis** [comment]
wait ... this branch does not have protobuf submodule yet ?

**Comment by therealssj** [comment]
it does have protobuf submodule and no its not auto generated. I made that change and made a mistake in the code

**Comment by olemis** [comment]
needed to fix this for testing #77
